docs-conceptual/azps-3.8.0/install-az-ps.md

```diff
@@ -60,39 +71,30 @@ Are you sure you want to install the modules from 'PSGallery'?
 
 Answer `Yes` or `Yes to All` to continue with the installation.
 
-The Az module is a rollup module for the Azure PowerShell cmdlets. Installing it downloads all of
-the available Azure Resource Manager modules, and makes their cmdlets available for use.
-
-> [!WARNING]
-> We do not support having both the AzureRM and Az modules installed for PowerShell 5.1 for Windows
-> at the same time. If you need to keep AzureRM available on your system, install the Az module for
-> PowerShell Core 6.x or later.
-
-First, [install PowerShell Core 6.x or later](/powershell/scripting/install/installing-powershell-core-on-windows)
-
-Then, from a PowerShell Core session, install the Az module for the current user only. This is the
-recommended installation scope.
-
-```powershell-interactive
-Install-Module -Name Az -AllowClobber -Scope CurrentUser
-```
-
 Installing the module for all users on a system requires elevated privileges. Start the PowerShell
 session using **Run as administrator** in Windows or use the `sudo` command on macOS or Linux:
 
 ```powershell-interactive
-Install-Module -Name Az -AllowClobber -Scope AllUsers
+if (Get-Module -Name AzureRM -ListAvailable) {
+    Write-Warning -Message 'Az module not installed. Having both the AzureRM and Az modules installed at the same time is not supported.'
+} else {
+    Install-Module -Name Az -AllowClobber -Scope AllUsers
+}
 ```
 
+The Az module is a rollup module for the Azure PowerShell cmdlets. Installing it downloads all of
+the available Az PowerShell modules, and makes their cmdlets available for use.
```

Comment:
we only download the stable Az PowerShell module. The ones in preview have to be installed independently.